src/main/webapp/stats.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Stats</title>
+  <link rel="stylesheet" href="/css/main.css">
+  
+  <script>
+    // Fetch stats and display them in the page.
```

Comment:
Please move the js code to a separate file. See the other html and js files in the repository for an example how that's done.